modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/redirect/button/RedirectButtonDDMFormFieldTemplateContextContributor.java

```diff
@@ -84,32 +83,34 @@ public Map<String, Object> getParameters(
 					RequestBackedPortletURLFactoryUtil.create(
 						ddmFormFieldRenderingContext.getHttpServletRequest());
 
+				Map<String, String[]> parameterMap = new HashMap<>();
```

Comment:
@JanisZhang Please rename the array in line 93 to `parameterPair` and rename this to `parameters`.